tests/python/gpu/test_tensorrt.py

```diff
@@ -0,0 +1,437 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Super helpful tests, thanks Clement.  It's ok to merge in a subset of these at a time, but I'd try to avoid pushing commented out code / too many todos upstream.

Reply:
I removed the commented out unit tests